### PR TITLE
Emphasize that dry run is default

### PR DESCRIPTION
**Pull request by ostromart**
Addresses https://github.com/istio/istio/issues/15322.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ go build -o $GOPATH/bin/iop ./cmd/iop.go
 The `iop` command supports the following flags:
 
 - `logtostderr`: log to console (by default logs go to ./iop.go).
-- `dry-run`: console output only, nothing applied to cluster or written to files (default is true for now).
+- `dry-run`: console output only, nothing applied to cluster or written to files (**default is true for now**).
```
Comment (ayj) [inline README.md]: s/for now//

File: README.md [hunk 2]
```diff
@@ -111,9 +111,10 @@ levels representing a child dependency, use the following command:
 iop manifest -o istio_manifests
 ```
 
+(Note that the default dry-run setting only writes the intended actions to log.)
 Use depth first search to traverse the created directory hierarchy when applying your YAML files. This is needed for
 correct sequencing of dependencies. Child manifest directories must wait for their parent directory to be fully applied,
-but not their sibling manifest directories.
+but not their sibling manifest directories.
```
Comment (ayj) [inline README.md]: extra space

Comment (ostromart) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (ostromart) [reply README.md]: I want to emphasize this is temporary - reworded.

**Pull request merged**